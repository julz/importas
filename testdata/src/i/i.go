package i

import (
	fff "fmt"
	"os"
)

func _() {
	fff.Println()
	_ = os.Stderr
}
