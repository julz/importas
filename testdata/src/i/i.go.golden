package d

import (
	autoscalingv1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
	special "knative.dev/serving/pkg/apis/serving/v1"
)

func foo() {
	autoscalingv1alpha1.Resource("")
	servingv1.Resource("")
}
